lib/realtime/tenants/connect.ex

```diff
@@ -88,10 +148,12 @@ defmodule Realtime.Tenants.Connect do
 
   ## Private functions
 
-  defp call_external_node(tenant_id) do
+  defp call_external_node(tenant_id, opts) do
     with tenant <- Tenants.Cache.get_tenant_by_external_id(tenant_id),
          {:ok, node} <- Realtime.Nodes.get_node_for_tenant(tenant) do
-      :erpc.call(node, __MODULE__, :connect, [tenant_id], 5000)
+      :erpc.call(node, __MODULE__, :connect, [tenant_id, opts], @erpc_timeout_default)
     end
   end
+
+  defp disconnect?(tenant_id), do: UsersCounter.tenant_users(tenant_id) <= 0
```

Comment:
the `tenant_users/1` function returns only non-negative values https://github.com/supabase/realtime/blob/4ec674e52a551e0e7a449fef6d6e8dea916e8e38/lib/realtime/user_counter.ex#L20

```suggestion
  defp disconnect?(tenant_id), do: UsersCounter.tenant_users(tenant_id) == 0
```